survey.html

```diff
@@ -93,10 +96,13 @@ <h1>Survey</h1>
             
                 <textarea id = life_meaning name="life_meaning" rows="5" cols="50" placeholder="Describe yourself here..."></textarea>
             </p> 
+
+            <p>
+              <button type="submit" form="form1" value="Submit"> Submit </button>
+            </p>
+          
          
           </form> 
-          
-          <button type="submit" form="form1" value="Submit"> Submit </button>
             
```

Comment:
Button should be inside form element
Form control sinside `<p>` as per lab instructions